ocrd_segment/extract_lines.py

```diff
@@ -109,7 +109,7 @@ def process(self):
             worksheet.write('B1', 'Text', bold)
             worksheet.write('C1', 'Status', bold)
             worksheet.write('D1', 'Image', bold)
-            symbols = 'ſ ꝛ aͤ oͤ uͤ æ œ Æ Œ ℳ  ç ę ë - ⸗ = Α α Β β ϐ Γ γ Δ δ Ε ε ϵ Ζ ζ Η η Θ θ ϑ Ι ι ' \
+            symbols = 'ſ ꝛ 𝔷 aͤ oͤ uͤ æ œ Æ Œ ℳ  ç ę ë - ⸗ = Α α Β β ϐ Γ γ Δ δ Ε ε ϵ Ζ ζ Η η Θ θ ϑ Ι ι ' \
```

Comment:
I am quite surprised by this. This character registers as `MATHEMATICAL FRAKTUR SMALL Z`. So outside of modern-day formula typesetting, this should not play any role. Of course there is a full set of Unicode codepoints with Fraktur glyphs for mathematical purposes, but my understanding was that historical OCR (and DTA/OCR-D transcription rules in particular) should not use these at all. Fraktur is but a font in most contexts, not a script. (I know some contemporary/computer-era Fraktur enthusiasts nevertheless use these for typesetting universally, but that's just a hack IMHO.) 

Conversely, if `𝔷`, then why not also `𝔐` or `𝕼`?

(I know, you might argue the same with the above `SCRIPT CAPITAL M`, which I found quite frequently in GT4HistOCR – as currency sign for marks. So you could equally say, where's `ℓ`, `ℛ`, `ℱ` etc? Answer: I am happy to include them here if someone can point me to actual use in historic texts.)

Reply:
Hah, sry, I did **not** mean to push this change, I share your skepticism towards (mis)using mathematical symbols for transcription of Fraktur but included it for discussion with our industrious transcribers. I'll force push it out of this PR.

But what I want to add is to change the default font. At least on our systems, the default is `Calibri` which is not very readable (`I` vs `l` etc.) and doesn't display `ꝛ` (r rotunda) correctly. From the list of built-in fonts, `Segoe UI Semibold` looks the most readable and has a fairly wide support of characters AFAICT.

Reply:
> Hah, sry, I did **not** mean to push this change, I share your skepticism towards (mis)using mathematical symbols for transcription of Fraktur but included it for discussion with our industrious transcribers. I'll force push it out of this PR.

I see. Thanks!

> But what I want to add is to change the default font. At least on our systems, the default is `Calibri` which is not very readable (`I` vs `l` etc.) and doesn't display `ꝛ` (r rotunda) correctly. From the list of built-in fonts, `Segoe UI Semibold` looks the most readable and has a fairly wide support of characters AFAICT.

Oh, sorry – I did not even notice `Calibri` was the default font for spreadsheets. Since I don't have any M$ fonts installed whatsoever, and my fallbacks all look good, I blindly assumed there was no problem in that regard. But judging by font preview on the web, `Segoe UI Semibold` also does not cover all the glyphs we want here (e.g. no `ℳ` or `⸗`).

Could we please try to find some widely available, ideally free font family?

Reply:
> Oh, sorry – I did not even notice Calibri was the default font for spreadsheets.

It is in Excel. In LibreOffice, the default font `Liberation Sans` displays all characters just fine.

> But judging by font preview on the web

There are fallbacks for those characters not in the font. Don't ask me how M$ font layering works but at least all characters are properly displayed. `I` vs `l` etc. are still a problem though.

> Could we please try to find some widely available, ideally free font family?

I have heard good things about https://www.pgdp.net/wiki/DP_Sans_Mono But the issue is more that we must work with what we have, having users install additional fonts is problematic. If we could bundle the font with the tables, while a waste of space, would be best for interoperability, haven't researched whether that's possible though, I doubt it though :(

Reply:
Understood. So we already have quite some complexity here: typical preinstalled fonts available in various OS, fallback/layering in OS' font servers, and even inter-operability (if someone picks up a file with Windows/Excel and makes edits, then sends it to me for confirmation/repairs, I'll see the `Calibri` fallback instead of the `Liberation Sans` default, and vice versa...)

Yes, I agree we should use some monospace font for our purposes here. But `DP Sans Mono` (like `DejaVu Sans Mono` and even `Liberation Sans Mono` or `Monospace Regular`) does not render superscript umlauts well. Also, I wonder whether sans-serif is really what we want. I found both `Latin Modern Mono` (from LaTeX) and `Ubuntu Mono` quite acceptable.

I agree we should try to work towards maximal compatibility with pre-installed fonts. But I have no idea how OSs configure family fallbacks. 

I think we could "bundle" a ttf or otf file along with the generated xlsx files for users who are willing to put up with right-clicking and installing (should work on all major platforms by now), though.

Reply:
I wish it were that easy but Windows is a terrible OS as we all know and agree on ;) This is what happens when I try "right-clicking and installing":

![image](https://user-images.githubusercontent.com/273367/105730705-efb9d600-5f2e-11eb-93a5-77b0c860008d.png)

And if I fail to install a font, I doubt my colleagues will succeed. I remember having to resort to a local Administrator account and hours of trial and error to get a functional monospace font installed.

Reply:
> I wish it were that easy but Windows is a terrible OS as we all know and agree on ;) This is what happens when I try "right-clicking and installing":
> 
> ![image](https://user-images.githubusercontent.com/273367/105730705-efb9d600-5f2e-11eb-93a5-77b0c860008d.png)
> 
> And if I fail to install a font, I doubt my colleagues will succeed. I remember having to resort to a local Administrator account and hours of trial and error to get a functional monospace font installed.

Wow, mind-boggling. I had no idea Windows is **that** much of a failure to this day.